custom_components/tuya_local/devices/qoto_05_water_valve.yaml

```diff
@@ -1,4 +1,7 @@
-name: QOTO 05 Water Valve
+name: Water Valve
```

Comment:
This and many of the entity names are still not following HA naming convention completely, as they have capitalised every word.
https://developers.home-assistant.io/blog/2022/07/10/entity_naming/